src/assets/main.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```

Comment:
It is cool that you use tailwind. It is a new instrument for me

Reply:
Nice instrument. My recommendation for you))